astropy/io/ascii/ipac.py

```diff
@@ -156,6 +149,14 @@ def process_keyword_value(val):
                     if val:
                         table_meta['comments'].append(val)
 
+    def get_col_type(self, col):
+        for (col_type_key, col_type) in self.col_type_list:
```

Comment:
Maybe the comments I suggested above should go here. Also, I don't see where this function is actually called, but I have to admit that I review this on a smartphone while traveling so I probably just overlooked it.